pkg/controller/clusterserviceversion/clusterserviceversion_controller.go

```diff
@@ -131,16 +164,65 @@ func (r *ReconcileClusterServiceVersion) Reconcile(request reconcile.Request) (r
 		return reconcile.Result{}, err
 	}
 
+	annotations := CSV.GetAnnotations()
+	if annotations == nil {
+		annotations = make(map[string]string)
+	}
+
+	//check if the CSV name, matches the RHM-Operator CSV name
+	if strings.Contains(CSV.GetName(), utils.CSV_NAME) {
```

Comment:
We'll need to check all CSVs, not just our operator. I think this is filtering to just ours.